docker/php/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:5.6-fpm
+
+# Basic packages
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+    && apt-get update \
+    && apt-get install -y \
+        git \
+        wget \
+        nodejs \
+    && rm -rf /var/lib/apt/lists/*
+
+# Composer
+RUN curl -k -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer \
+    && chmod +x /usr/local/bin/composer \
+    && composer global require "hirak/prestissimo:^0.3"
+
+# PHP packages + config
+RUN curl -L -o /usr/local/bin/docker-php-pecl-install https://raw.githubusercontent.com/helderco/docker-php/master/template/bin/docker-php-pecl-install \
+    && chmod +x /usr/local/bin/docker-php-pecl-install
+
+RUN docker-php-ext-install -j$(nproc) \
+    pdo \
+    pdo_mysql
+
+# Install xdebug
+RUN pecl install xdebug-2.5.5 \
+    && docker-php-ext-enable xdebug
+
+COPY php.ini /usr/local/etc/php/conf.d/custom.ini
+
+# Set default user + dir
+USER www-data
```

Comment:
I only use `USER www-data` to default connect to container with `www-data` user. Otherwise we just have to specify user : `docker-compose exec --user www-data php bash`. I'll remove the line since `www-data`is created by default.

It give: 
```bash
# cat /etc/passwd | grep www-data
www-data:x:33:33:www-data:/var/www:/usr/sbin/nologin
```

I'll add alias ;-)

Reply:
I tried also without this clause... I was surprise to see that even without this (which means root?! right??). I still had rights issue. Maybe root is not the 1000 user? It's something weird with docker I don't explain for now.

Btw it's probably fixed on dockermac because they run it inside a VM that have no special rights on.

Reply:
It may be because of ACL I used on my cache directory (well it's probably because of that). As my user (nek) is 1000 on my computer, forcing www-data to 1000 inside the docker fix the issue ("www-data" use the same id as "nek" on my computer and all is good).

This fix may be an issue if somebody comes out with a user with id 1001 :')... I don't have a better solution for now.